### ISSUE TITLE
Add published WikiPathways examples to Zenodo, from Tumblr

### ISSUE THREAD

**Issue by khanspers** [wikipathways/wikipathways-help#11]
Go through Tumblr archive to find papers where specific WPs were used in figures. Upload to Zenodo WP community.

**Comment by khanspers** [comment]
Making a list here for now:

- https://www.wikipathways.org/index.php/Pathway:WP702, used in [Jennen et al](https://cris.maastrichtuniversity.nl/ws/files/72447358/evelo_2010_biotransfromation_pathway_maps_in_wikipathways.pdf), fig 2. 2010 Oct.
- https://www.wikipathways.org/index.php/Pathway:WP688, used in [Jennen et al](https://cris.maastrichtuniversity.nl/ws/files/72447358/evelo_2010_biotransfromation_pathway_maps_in_wikipathways.pdf), fig 1. 2010 Oct.
- https://www.wikipathways.org/index.php/Pathway:WP1269, used in [Coort et al](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2593012/), fig 1. I think an earlier version of this pathway was used in the paper. 2008 Nov 26.
- https://www.wikipathways.org/index.php/Pathway:WP336, used in [Coort et al](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2593012/), fig 2. 2008 Nov 26.
- https://www.wikipathways.org/index.php/Pathway:WP65, used in [Coort et al](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2593012/), fig 3. 2008 Nov 26.
- https://www.wikipathways.org/index.php/Pathway:WP2036, used in [Bhattacharjee et al](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3357548/), fig 3. 2012 May 9. (NetPath)
- https://www.wikipathways.org/index.php/Pathway:WP2037, used in [Radhakrishnan et al](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3421022/), fig 1. 2012 Jun 9. (NetPath)
- https://www.wikipathways.org/index.php/Pathway:WP2324, used in [Soman et al](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3590362/), fig 1. 2012 Nov 19. (NetPath)
- https://www.wikipathways.org/index.php/Pathway:WP1982 used in [Daemen et al](https://link.springer.com/article/10.1007/s12263-013-0342-x), fig 1. 21 March 2013.
- https://www.wikipathways.org/index.php/Pathway:WP2374 used in [Dey et al](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3660694/), fig 1. 2013 Jun. (NetPath)
- https://www.wikipathways.org/index.php/Pathway:WP707 (earlier version of), used in [van Delft et al](https://doi.org/10.1093/toxsci/kfs250), fig 6. 13 August 2012.
- https://www.wikipathways.org/index.php/Pathway:WP2355: [Subbannayya et al](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3889255/), fig 1. 2013 Mar 16. (NetPath)

**Comment by DeniseSl22** [comment]
@khanspers; has this information on citations been added to the new website already? If not, I'm willing to take care of that (or close this issue).

**Comment by khanspers** [comment]
Yes, all of the citations listed here have already been added to the new site. There's a GitHub action that runs weekly to fetch new citations from PMC: https://github.com/wikipathways/wikipathways-database/actions/workflows/scheduled_citedin_update.yml.

This particular issue was about adding published pathway figures from WP to Zenodo. Not sure it's relevant anymore.

**Comment by mkutmon** [comment]
We should discuss this in the next meeting if this is still needed.
